docs/ref-info/glossary-uid.md

```diff
@@ -205,6 +214,10 @@ sidebar_position: 10
 <dt class="jump-anchor" id="gl-private-operator-service">Private Operator Service</dt>
 <dd>A private instance of the Operator Service, run by a <a href="#gl-private-operator">Private Operator</a>.</dd>
 
+<dt class="jump-anchor" id="gl-public-key">Public key</dt>
+<dd>In the context of client-side token generate (CSTG), the public key is one of the two values issued to publishers as their UID2 credentials. For details, see <a href="../getting-started/gs-credentials#subscription-id-and-public-key">Subscription ID and Public Key</a>.</dd>
```

Comment:
for the record, subscription ID is just an arbitrary ID associated with the public key (and its associated private key), it doesn't have much else meaning. (we internally use it to find out which UID2 site the ID/public key belongs to)

Reply:
Thank you! Good to know.